lib/datastore/query.js

```diff
@@ -75,7 +75,16 @@ function Query(namespace, kinds) {
 }
 
 /**
+ * @param {boolean=} autoPaginateVal - Have pagination handled automatically.
+ *     Default: true.
  * @return {module:datastore/query}
+ *
+ * @example
+ * // Have pagination handled automatically
+ * var autoPaginated = query.autoPaginate();
```

Comment:
Haha, sure. Do you think we should overhaul most of the query examples then? They kind of all resemble that format..

https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/datastore/query.js#L131
https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/datastore/query.js#L149
https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/datastore/query.js#L171